### PR TITLE
test inmantals against iso5-stable

### PR DESCRIPTION
**Pull request by sanderr**

### CODE REVIEW

File: server/tox.ini [hunk 1]
```diff
@@ -18,6 +18,7 @@ deps=
     inmlatest: inmanta-core
     inmmaster: git+https://github.com/inmanta/inmanta-core.git
     inmiso4-stable: git+https://github.com/inmanta/inmanta-core.git@iso4-stable
+    inmiso5-stable: git+https://github.com/inmanta/inmanta-core.git@iso5-stable
```
Comment (arnaudsjs) [inline server/tox.ini:21]: * We need to add `iso5-stable` to `tox.envlist` as well.
* `tox.envlist` indicates that the tests run using `python3.6` and `python3.8`. I don't think that's accurate.

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

File: server/tox.ini (hunk 1 above)
Comment (sanderr) [reply server/tox.ini:21]: Think I fixed both.

File: server/tox.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = pep8,py{36,38}-inm{master,iso4-stable,latest}
+envlist = pep8,py36-inmiso4,py39-inm{master,iso{4,5}-stable,latest}
```
Comment (arnaudsjs) [inline server/tox.ini:2]: Why also run `iso4` using `python39`?

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

File: server/tox.ini (hunk 2 above)
Comment (sanderr) [reply server/tox.ini:2]: Good point, it's not OSS so I guess 3.6 suffices.

**Review by arnaudsjs — APPROVED** [review]

**Pull request merged**